wiki/Article_Styling_Criteria/en.md

```diff
@@ -570,40 +487,32 @@ Installing osu! is easy. First, download the installer from the download page.
 Then locate the installer and run it.
 ```
 
-There is no way to float an image or have text wrap around an image. If the image is inline with the raw text, the image is parsed inline with the text.
+*Note: It is not possible to float an image or have text wrap around it. If the image is placed inline with the text, the image will be displayed inline with the text.*
 
 #### Alternate and title text
 
-All images should have an alternate text. This will ensure that something will be displayed even if the image fails to load. Title text is optional and could be the same as the alt text if needed.
+All images should have alternate text. This will ensure that something will be displayed even if the image fails to load. Title text is optional and could be the same as the alternate text if needed.
 
 #### Captions
 
-Images are automatically given captions if they fulfill these rules:
+Images are automatically given captions if they fulfill these conditions:
 
 1. The image is by itself.
 2. The image is not inside a heading.
 3. The image has title text.
 
-Captions are assumed via the title text. Image captions must be plain text and never contain formatting nor link markup. Images with captions are also centred with the image on the website.
+Captions are assumed via the title text. Image captions must be plain text without the. Images with captions are also centred with the image on the website.
 
 #### Max image width
 
-The osu! wiki will give every image a max-width, being the width of the article body (680 pixels). This means when an image's width is more than 680 pixels, you should consider downscaling it to 680 pixels to limit wasted digital space.
+The osu! wiki gives every image a max-width, being the width of the article body (680 pixels). When an image's width is more than 680 pixels, consider downscaling it.
 
 #### Annotating images
 
 When annotating images, use *Exo 2 light*. For Chinese, Korean, Japanese characters, use *Microsoft YaHei*.
 
 Try to limit writing text in images, as it is difficult for translators (and other editors) to edit them.
 
-##### Translating annotated images
```

Comment:
Any reason removing this? It explains a proper format.

This can surely be used when taking pictures of the osu! client, which has locales/translations.